tools/proofers/preview.js

```diff
@@ -895,12 +956,42 @@ $(function () {
             if (viewMode !== "show_tags") {
                 colorString = colorString0 + '>$1</span>';
             }
-            // sub- and super-scripts
-            txt = txt.replace(/_\{([^\}]+)\}/g, '<span class="sub"' + colorString);
-            txt = txt.replace(/\^\{([^\}]+)\}/g, '<span class="sup"' + colorString);
-            // single char superscript -  any char except {
-            // do not allow < incase it's a tag which would screw up
-            txt = txt.replace(/\^([^\{<])/g, '<span class="sup"' + colorString);
+
+            // process sub- and super-scripts
+            function ssProcess(text) {
+                text = text.replace(/_\{(.+?)\}/g, '<span class="sub"' + colorString);
+                text = text.replace(/\^\{(.+?)\}/g, '<span class="sup"' + colorString);
+                // single char superscript -  any char except {
+                // do not allow < incase it's a tag which would screw up
```

Comment:
should we just encode and cover all issues?

Reply:
It's a long time ago, but I think this was meant to avoid two overlapping html markups which would produce a nonsensical result, but I'll have another look at it.